src/components/ProductList/ProductList.js

```diff
@@ -39,8 +40,11 @@ export const ProductList = () => {
                   alt={filterProduct.title}
                   class="img"
                 />
-                { wishlistState.find(wishlistItem => wishlistItem._id === filterProduct._id) ? (
-                  <span key={filterProduct._id}
+                {wishlistState.find(
+                  (wishlistItem) => wishlistItem._id === filterProduct._id
```

Comment:
doesn't make any sense writing same thing again....you can make isInCart and isInWishlist function in utils and then wherever you want just call them by sending required argument